### PR TITLE
feat: Attachment view improvements

### PR DESCRIPTION
**Pull request by iamsivin**
# Pull Request Template

## Description

- Shows attachment sender name and time stamp
- Adds file name
- Adds total count and active count (eg: 2/10)
- Adds download button

Fixes https://linear.app/chatwoot/issue/CW-1873/attachment-preview

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested
https://www.loom.com/share/adc09af15cf4491080224a65199f627c?sid=c46b0ed1-d39e-4e5c-90dd-54781f56e856

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by nithindavid** [comment]
@iamsivin Any reason why we are not showing time stamp here?
image

**Comment by iamsivin** [comment]
> @iamsivin Any reason why we are not showing time stamp here?
> 
> image
> 
> 

We are showing it there. Can check does the value is there or not?

File: app/javascript/dashboard/components/widgets/conversation/components/GalleryView.vue [hunk 1]
```diff
@@ -158,27 +239,130 @@ export default {
...
 .gallery-modal--wrap {
-  display: flex;
-  flex-direction: row;
   align-items: center;
-  width: inherit;
+  display: flex;
+  flex-direction: column;
   height: inherit;
+  width: inherit;
+
```
Comment (nithindavid) [inline app/javascript/dashboard/components/widgets/conversation/components/GalleryView.vue:262]: Any way we could reduce the css here? We could use `text-block-title` for 14px bold texts, not using flex centers when not needed?

**Review by nithindavid — COMMENTED** [review]
Overall looks good and the attachment view feels good.

**Comment by nithindavid** [comment]
> > @iamsivin Any reason why we are not showing time stamp here?
> > image
> 
> We are showing it there. Can check does the value is there or not?

@iamsivin It does have value when I click the message right after uploading it. After refresh this is working as expected.

**Comment by iamsivin** [comment]
> Overall looks good and the attachment view feels good.

Thanks for testing the PR. I have updated the PR can you take a look?

**Review by nithindavid — APPROVED** [review]

**Pull request merged**